.github/sync-repo-settings.yaml

```diff
@@ -15,3 +15,4 @@ branchProtectionRules:
     - 'Samples - Python 3.8'
     - 'Samples - Python 3.9'
     - 'Samples - Python 3.10'
+    - 'GCF Owl Bot / OwlBot Post Processor'
```

Comment:
```suggestion
    - 'OwlBot Post Processor'
```
To match the default required checks [here](https://github.com/googleapis/repo-automation-bots/blob/main/packages/sync-repo-settings/src/required-checks.json#L46)